package.json

```diff
@@ -38,9 +38,9 @@
     ]
   },
   "scripts": {
-    "build": "yarn clean && yarn build:scss && yarn build:css && yarn build:html && yarn copy-files",
+    "build": "yarn clean && yarn build:scss && yarn build:css && yarn lint:copyright --fix lib/css/* && yarn build:html && yarn copy-files",
     "build:scss": "cpx \"./src/**/*.scss\" ./lib/scss",
-    "build:css": "node ./scripts/generateCss.js src lib/css && yarn postcss-processing && yarn lint:copyright --fix lib/css/*",
+    "build:css": "node ./scripts/generateCss.js src lib/css",
```

Comment:
so `postcss-processing` is not needed anymore? Remove from scripts too

Reply:
Good catch! Removed.